app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/FullDetailsFragment.java

```diff
@@ -1199,6 +1201,7 @@ public void onResponse(List<BaseItemDto> response) {
                     return;
                 }
 
+                interactionTracker.getValue().notifyStartSession(item, response);
```

Comment:
I think it might be better to call this from the actual video player when it starts playback